#self.center, self.half_of_size_x, self.half_of_size_y, self.matrix, self.path,
#Recording the start and goal of the navigation

#lat lng of center position
geographic_msgs/GeoPoint center

#map pixels
int64 half_of_size_x
int64 half_of_size_y

#map
int64 width
int64 height
uint8[] matrix

#path
int64[] path

---

std_msgs/Empty response